### PR TITLE
Added fips_compliant for enabling fips

### PR DESCRIPTION
**Pull request by gauravpbankar**
Signed-off-by: gauravpbankar <gauravpbankar@gmail.com>

1. Added fips_compliant variable in playbook code

2. added fips_compliant in ocp-power-vs code that will be prapogate in playbook , below is the link for the same.
https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/378/files 
 
3. also updated variable name from fips to fips_compliant in defaults, examples, README file.

### CODE REVIEW

**Review by cs-zhang — DISMISSED** [review]
/lgtm

File: playbooks/roles/ocp-config/README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Role Variables
 | proxy_url                   | no       | ""             | Proxy url eg: http://[user:passwd@]server:port (NA when setup_squid_proxy: true)|
 | no_proxy                    | no       | ""             | Comma seperated string of domains/cidr to exclude proxy |
 | enable_local_registry       | no       | false          | Set to true to enable usage of local registry for restricted network install |
+| fips_compliant              | no       | false          | Set to true to enable usage of fips for ocp deployment |
```
Comment (yussufsh) [inline playbooks/roles/ocp-config/README.md]: Minor comments.
fips->FIPS
ocp->OCP

**Comment by Prajyot-Parab** [comment]
@gauravpbankar please amend your commit message (it has been signed off twice + has squashed commit msg too)

**Comment by gauravpbankar** [comment]
> @gauravpbankar please amend your commit message (it has been signed off twice + has squashed commit msg too)

Done

**Review by yussufsh — APPROVED** [review]

**Review by Prajyot-Parab — APPROVED** [review]
/lgtm

**Pull request merged**